src/LiveDevelopment/Inspector/Inspector.js

```diff
@@ -116,7 +116,8 @@ define(function Inspector(require, exports, module) {
      */
     function _send(method, signature, varargs) {
         if (!_socket) {
-            console.log("You must connect to the WebSocket before sending messages.");
```

Comment:
You can revert this change, since we don't ever use this code path in the Browser.